requirements/edx/base.txt

```diff
@@ -119,7 +119,7 @@ django_debug_toolbar==1.3.2
 
 # Used for testing
 astroid==1.3.8
-bok-choy==0.4.6
+#bok-choy==0.4.6
```

Comment:
is the plan to get https://github.com/edx/bok-choy/pull/118 merged in, make a new version, then use that version number here?